DakeAndDevileCorps/Screens/Store/VC/StoreInformationTableViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  StoreInformationTableViewCell.swift
+//  DakeAndDevileCorps
+//
+//  Created by 최동권 on 2022/07/19.
+//
+
+import UIKit
+
+class StoreInformationTableViewCell: UITableViewCell {
+    
+    static let identifier = "StoreInformationTableViewCell"
+    
+    @IBOutlet weak var operationStatus: UILabel!
+    @IBOutlet weak var todayOperationTime: UILabel!
+    @IBOutlet weak var productCategories: UILabel!
+    @IBOutlet weak var address: UILabel!
+    @IBOutlet weak var phoneNumber: UILabel!
+    @IBOutlet weak var operationTime: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    func setUpperData(isOperation: Bool, todayOperationTime: String, productCategories: String) {
+        operationStatus.text = isOperation ? "영업중" : "영업 종료"
+        operationStatus.textColor = isOperation ? .systemGreen : .systemRed
+        self.todayOperationTime.text = todayOperationTime
+        self.productCategories.text = productCategories
+        self.productCategories.textColor = .secondaryLabel
```

Comment:
p5;

self.를 써도 안써도 같은 의미라서 상관없지만 통일성을 지켜주시면 좋을 거 같아요!
```suggestion
        operationStatus.text = isOperation ? "영업중" : "영업 종료"
        operationStatus.textColor = isOperation ? .systemGreen : .systemRed
        todayOperationTime.text = todayOperationTime
        productCategories.text = productCategories
        productCategories.textColor = .secondaryLabel
```

Reply:
다른 이름으로 명명하면 혼란이 생기지 않겠지만,
같은 이름으로 만들었을 때의 혼란스러움보다, 해당 이름이 주는 적절성이 더 클 때는
self를 사용하여 표현해도 좋을 것 같습니다.